aviatrix/resource_aviatrix_vpn_user.go

```diff
@@ -129,6 +258,20 @@ func resourceAviatrixVPNUserDelete(d *schema.ResourceData, meta interface{}) err
 
 	log.Printf("[INFO] Deleting Aviatrix VPNUser: %#v", vpnUser)
 
+	if _, ok := d.GetOk("profiles"); ok {
+		log.Printf("[INFO] Found profiles: %#v", d.Get("profiles"))
+		for _, profileName := range d.Get("profiles").([]interface{}) {
+			profile := &goaviatrix.Profile{
+				Name: profileName.(string),
+			}
+			profile.UserList = append(profile.UserList, vpnUser.UserName)
+			err := client.DetachUsers(profile)
```

Comment:
I think this is automatically done in the controller

Reply:
Removed it.